web_src/js/components/RepoActionView.vue

```diff
@@ -374,6 +400,27 @@ export function initRepositoryActionView() {
   padding: 10px;
 }
 
+.job-artifacts-title {
+  font-size: 110%;
```

Comment:
```suggestion
  font-size: 18px;
```

Should be about equal, and I do prefer integer values for font size.